<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bragg List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bragg Points used for Currat-Axe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="BraggList_listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="BraggList_H_spinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.998999999999995</double>
         </property>
         <property name="maximum">
          <double>99.998999999999995</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="BraggList_K_spinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.998999999999995</double>
         </property>
         <property name="maximum">
          <double>99.998999999999995</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="BraggList_L_spinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.998999999999995</double>
         </property>
         <property name="maximum">
          <double>99.998999999999995</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>L</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>H</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>K</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="BraggList_add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BraggList_delete_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BraggList_deleteAll_button">
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>BraggList_listView</tabstop>
  <tabstop>BraggList_H_spinBox</tabstop>
  <tabstop>BraggList_K_spinBox</tabstop>
  <tabstop>BraggList_L_spinBox</tabstop>
  <tabstop>BraggList_add_button</tabstop>
  <tabstop>BraggList_delete_button</tabstop>
  <tabstop>BraggList_deleteAll_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
